src/components/Footer/Footer.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
-import { useOktaAuth } from '@okta/okta-react';
-import { useHistory } from 'react-router-dom';
-import { Button } from 'antd';
```

Comment:
Good eye for cleaning up the unused imports in this component upon making this change!